### PR TITLE
Remove title

### LINKED ISSUE #158: Remove the title

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
Fixes #158

### CODE REVIEW

**Comment by aappling-usgs** [comment]
there must be a call or two to `updateTitle` somewhere, right? i think those will also need to be deleted.

**Comment by ldecicco-USGS** [comment]
Bonus:
Fixes #151

Figured it out via:
https://stackoverflow.com/questions/15178700/javascript-functions-and-default-parameters-not-working-in-ie-and-chrome

File: js/map_utils.js [hunk 1]
```diff
@@ -185,10 +186,22 @@ function documentCategorySwitch(category, prevCategory, action) {
 }
 
 
+function highlightCircle(currentCircle) {
```
Comment (ldecicco-USGS) [inline js/map_utils.js]: I don't know what this is...

Comment (lindsaycarr) [reply js/map_utils.js]: it's what makes the circle go from slightly see-through to solid when you hover

Comment (lindsaycarr) [reply js/map_utils.js]: not sure how it is being added here though...they should've been there the whole time. Same with `unhighlightCircle` below

Comment (ldecicco-USGS) [reply js/map_utils.js]: but how did it get in my pull request? probably something I did on the merge conflict

File: js/styles.js [hunk 2]
```diff
@@ -29,14 +29,22 @@ function emphasizeCounty(selection) {
     .classed("emphasize-county", true);
 }
 
-function foregroundState(selection, scale = 1) {
+function foregroundState(selection, scale) {
```
Comment (lindsaycarr) [inline js/styles.js]: so were these the trick to #151? Setting a default to the functions?

Comment (ldecicco-USGS) [reply js/styles.js]: Seems like it

File: js/build_map.js [hunk 3]
```diff
@@ -91,14 +95,12 @@ function prepareMap() {
 
   // create map title
   svg.append("text")
-    .attr("id", "maptitle")
     .attr("x", chart_width/2)
     .attr("y", chart_height*0.10); // bring in 10% of chart height
```
Comment (aappling-usgs) [inline js/build_map.js]: looks like the merge conflict resolution brought 3 of these 4 lines back in

File: js/map_utils.js [hunk 4]
```diff
@@ -185,10 +186,22 @@ function documentCategorySwitch(category, prevCategory, action) {
...
+      origNode = orig.node();
+  var duplicate = d3.select(origNode.parentNode.appendChild(origNode.cloneNode(true), 
+                                                            origNode.nextSibling));
+                                                            
+  // style duplicated circles sitting on top
+  duplicate
+    .classed('circle-duplicate', true)
+    .style("pointer-events", "none")
+    .style("opacity", 1); // makes the duplicate circle on the top
+}
```
Comment (aappling-usgs) [inline js/map_utils.js]: i don't think lines 190 through 200 should be in here following the merge of my PR

File: js/map_utils.js [hunk 5]
```diff
@@ -185,10 +186,22 @@ function documentCategorySwitch(category, prevCategory, action) {
...
 
-function updateTitle(category) {
-  d3.select("#maptitle")
-    .text("Water Use Data for " + activeView + ", 2015, " + category);
+function unhighlightCircle() {
+  d3.select('.circle-duplicate')
+    .remove(); // delete duplicate
```
Comment (aappling-usgs) [inline js/map_utils.js]: The unhighlightCircle function should look pretty different from both of these. Should be: https://github.com/USGS-VIZLAB/water-use-15/blob/master/js/circles.js#L100-L105
```js
function unhighlightCircle() {

  map.select('circle#wu-highlight')
    .classed('hidden', true);

}
```

File: js/map_utils.js (hunk 4 above)
Comment (aappling-usgs) [reply js/map_utils.js]: oh! also, the functions highlightCircle and unhighlightCircle should now live in circles.js, not here in map_utils.js

File: js/map_utils.js (hunk 1 above)
Comment (aappling-usgs) [reply js/map_utils.js]: yes, i think this is a merge thing, probably because i moved highlightCircle and unhighlightCircle to circles.js

**Comment by aappling-usgs** [comment]
i think you still need to delete lines 97-99 of build_map.js

File: js/build_map.js [hunk 6]
```diff
@@ -91,14 +91,12 @@ function prepareMap() {
...
   
   /** Initialize title and URL **/
```
Comment (ldecicco-USGS) [inline js/build_map.js]: So...these comments need to be deleted?

**Pull request merged**